src/mpi_t/events_impl.c

```diff
@@ -124,6 +129,7 @@ void MPIR_T_event_instance(int event_index, MPI_T_cb_safety cb_safety, void *dat
 
     return;
 }
+#endif /* HAVE_MPIT_EVENTS */
```

Comment:
I don't quite understand why do we need to `#ifdef` out the entire function. Isn't `#ifdef` out the body sufficient?

Reply:
After adding macro wrapper, changed the ifdef to just do the body of the function here.